libs/cli/src/generators/challenge/generator.ts

```diff
@@ -85,7 +88,7 @@ export async function challengeGenerator(tree: Tree, options: Schema) {
     `./docs/src/content/docs/challenges/${options.docRepository}`,
     {
       tmpl: '',
-      projectName: names(options.name).name,
+      projectName: appProjectName,
```

Comment:
to be more consistent, you should rename `projectName` into `appProjectName` (dont forget to do it inside the doc file.